packages/design-tokens/src/index.ts

```diff
@@ -0,0 +1,205 @@
+export const radiusFull = '999em';
```

Comment:
it may be nice to add comments for each section like: 
```
// Colors
export const colorBackgroundWarning = '#fbede7';
export const colorBorderDanger = '#b4362f';
export const colorBorderDisabled = '#f4f4f5';
```

Reply:
yeah I was going to come back and separate content out to make it easier to read. I can do that now